app/main.py

```diff
@@ -1,3 +1,67 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, new: Distance | float | int) -> Distance:
+        if isinstance(new, Distance):
+            return Distance(self.km + new.km)
+        return Distance(self.km + new)
+
+    def __iadd__(self, new: Distance | float | int) -> Distance:
+        if isinstance(new, Distance):
+            self.km += new.km
+            return self
+        self.km += new
+        return self
+
+    def __mul__(self, new: float | int) -> Distance:
+        return Distance(self.km * new)
+
+    def __truediv__(self, new: int | float) -> Distance | None:
+        if new:
+            return Distance(round(self.km / new, 2))
+        return
+
+    def __lt__(self, new: Distance | float | int) -> bool:
+        if isinstance(new, Distance) and self.km < new.km:
```

Comment:
Please try to simplify this method. The result of the comparison will already return a boolean value.